operator/database/kinds/backups/s3/backup/command.go

```diff
@@ -0,0 +1,53 @@
+package backup
+
+import (
+	"strconv"
+	"strings"
+)
+
+func getBackupCommand(
+	timestamp string,
+	bucketName string,
+	backupName string,
+	certsFolder string,
+	accessKeyIDPath string,
+	secretAccessKeyPath string,
+	sessionTokenPath string,
+	region string,
+	endpoint string,
+	dbURL string,
+	dbPort int32,
+) string {
+
+	backupCommands := make([]string, 0)
+	if timestamp != "" {
+		backupCommands = append(backupCommands, "export "+backupNameEnv+"="+timestamp)
+	} else {
+		backupCommands = append(backupCommands, "export "+backupNameEnv+"=$(date +%Y-%m-%dT%H:%M:%SZ)")
+	}
```

Comment:
This code and the timestamp parameter is only for making the function testable?
I think it would be better to adjust the assertion in the test in order to avoid accidential complexity in the production code.

Reply:
No this part of the code is so that the timestamp is done with the same command that does the backup itself, and the if only separates if a defined backup-name is given by the command, defaults to the timestamp logic.